ci/environment.yml

```diff
@@ -37,5 +37,3 @@ dependencies:
   - geopandas
   - xarray
   - asv
-  - pip:
-    - git+https://github.com/philipc2/pyfma.git # fix until https://github.com/nschloe/pyfma/pull/17 is merged
```

Comment:
This file also has `gmpy2` as a dependency

Reply:
Good catch!

Reply:
Similarly the `docs.yml` and `asv.yml`, too.